### DISCUSSION TITLE
Segmentation fault (core dumped) - Fedora Kinolite

### DISCUSSION THREAD

**Discussion by tjbbjt** [evilsocket/opensnitch#1072] [Q&A]
Testing out Fedora Kinolite, rpm's and dependencies install fine via rpm-ostree but opensnitch-ui crashes upon launch with error "Segmentation fault (core dumped)"

Any suggestions?

**Comment by gustavo-iniguez-goya** [comment]
Hi @tjbbjt ,

Sorry for the delay answering this. Could you launch it from a terminal and post the logs?

`~ $ opensnitch-ui --debug`

**Reply by tjbbjt** [reply]
It must  have been a pre-release bug, I tried Kinoite again just now in a virtual environment and it works fine. 

Working fine in Fedora 40 pre-release too. 👍
